iceoryx_hoofs/testing/include/iceoryx_hoofs/testing/error_reporting/testing_error_handler.hpp

```diff
@@ -62,28 +82,28 @@ class TestErrorHandler : public iox::er::ErrorHandlerInterface
 
     /// @brief Indicates whether there was a panic call previously.
     /// @return true if there was a panic call, false otherwise
-    bool hasPanicked() const;
+    bool hasPanicked() const noexcept;
```

Comment:
@elBoberido What about this noexcept? Is this really useful? I had some objections here before but maybe they got lost. Have to check.

Reply:
I removed them from the `onPanic` and other hooks. The `hasPanicked` method cannot throw and has no hooks and therefore `noexcept` communicates exactly what the method does.